app/presenters/unauthorized_presenter.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+class UnauthorizedPresenter
+  def initialize(params)
+    @params = params
```

Comment:
@awead See my comment below: https://github.com/aic-collections/aicdams-lakeshore/pull/347#issuecomment-275953096 this could be any unauthorized page for GenericWork. Not sure if I'm in the right context here though...